packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -1152,6 +1152,29 @@ export const audiusBackend = ({
       console.error(getErrorMessage(err))
       return []
     }
+  } 
+  
+  // trackIds, start, end
+  async function getUserListenCountsMonthly(
+    currentUserId: number,
+    startTime: string,
+    endTime: string,
+  ) {
+    try {
+      const userListenCountsMonthly = await withEagerOption(
+        {
+          normal: (libs) => libs.User.getUserListenCountsMonthly,
+          eager: DiscoveryAPI.getUserListenCountsMonthly
+        },
+        encodeHashId(currentUserId),
+        startTime,
+        endTime
+      )
+      return userListenCountsMonthly
+    } catch (e) {
+      console.error(e)
```

Comment:
Let's use the getErrorMessage util here. It essentially casts the e: any type to string and handles all the potential error types